src/main/java/ru/sbt/mipt/oop/home/SmartHome.java

```diff
@@ -0,0 +1,40 @@
+package ru.sbt.mipt.oop.home;
+
+import ru.sbt.mipt.oop.event_handlers.ClosedHallDoorEvent;
+import ru.sbt.mipt.oop.event_handlers.SensorEvent;
+
+import java.util.Collection;
+
+public class SmartHome {
+
+    Collection<Room> rooms;
+
+    public Collection<Room> getRooms() {
+        return rooms;
+    }
+
+    public SmartHome(Collection<Room> rooms) {
+        this.rooms = rooms;
+    }
+
+    public void addRoom(Room room) {
+        rooms.add(room);
+    }
+
+    public void handleLightEvent(SensorEvent event) {
+        for (Room room : this.getRooms()) {
+            room.handleLightEvent(event);
+        }
+    }
+    public void handleDoorEvent(SensorEvent event) {
+        for (Room room : this.getRooms()) {
+            if (room.getName().equals("hall")){
```

Comment:
https://github.com/iagoshkov/smart-home-2020/blob/master/STUDENT_GUIDE.md#%D0%B4%D0%B5%D0%BB%D0%B5%D0%B3%D0%B8%D1%80%D1%83%D0%B9%D1%82%D0%B5-%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D1%83-%D1%81%D0%BE%D0%B1%D1%8B%D1%82%D0%B8%D0%B9-%D0%BE%D1%82%D0%B4%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%BC-%D0%BA%D0%BB%D0%B0%D1%81%D1%81%D0%B0%D0%BC